test/spec/quicktest_spec.rb

```diff
@@ -13,9 +13,43 @@
 
 require_relative "#{CONFIG['app_name']}/quicktest_custom_spec.rb"
 
+def new_driver(webdriver_url)
+  if @config.key?('proxy') and !webdriver_url.include? "browserstack"
+    proxy = Selenium::WebDriver::Proxy.new(
+      :http     => @config['proxy'],
+      :ftp      => @config['proxy'],
+      :ssl      => @config['proxy']
+    )
+    caps = Selenium::WebDriver::Remote::Capabilities.chrome(:proxy => proxy)
+    dr = Selenium::WebDriver.for(:chrome, :desired_capabilities => caps , :url => @config['chromedriver_url'])
+  else
+    dr =  Selenium::WebDriver.for(:remote, url: webdriver_url, desired_capabilities: browser_capabilities)
+  end
+  return dr
+end
+
+def navigate(url)
+    new_url = url
+    if defined? $caller_name and $caller_name.length > 0
```

Comment:
maybe we should use this opportunity to refactor quick tests and avoid repeated code. I have noticed that many functions were just copy/pasted from the original spec file, which means that any bug or change is not duplicated too. we needa way to REFER to tests without duplicating them